src/System.Private.ServiceModel/src/System/ServiceModel/Channels/SocketConnection.cs

```diff
@@ -486,7 +522,7 @@ private void ThrowIfNotOpen()
             {
                 throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(
                     ConvertObjectDisposedException(new ObjectDisposedException(
-                    GetType().ToString(), SR.SocketConnectionDisposed), TransferOperation.Undefined));
+                    GetType().ToString(), SR.Format(SR.SocketConnectionDisposed)), TransferOperation.Undefined));
             }
```

Comment:
SR.Format isn't needed when there are no parameters for the string resource

Reply:
👍